contexts/credentials/v2

```diff
@@ -112,9 +112,9 @@
       }
     },
 
-    "JsonSchema2023": {
+    "JsonSchemaCredential": {
       "@id":
-        "https://w3.org/2018/credentials#JsonSchema2023",
+        "https://w3.org/2018/credentials#JsonSchemaCredential",
```

Comment:
Since no scoped context is needed here, this can be simplified to:

```
    "JsonSchemaCredential": "https://w3.org/2018/credentials#JsonSchemaCredential",
```

Reply:
thanks, updated